pkg/kubelet/kubelet.go

```diff
@@ -1290,6 +1291,9 @@ func (kl *Kubelet) initializeModules() error {
 		}
 	}
 	if kl.enablePluginsWatcher {
+		// Adding Registration Callback function for CSI Driver
+		kl.pluginWatcher.AddHandler("CSIPlugin", csi.RegistrationCallback)
```

Comment:
This could be used from API instead of harcoding https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/pluginregistration/v1alpha1/constants.go#L20

Reply:
yep, will do in the follow up PR.